giskard_vision/core/dataloaders/base.py

```diff
@@ -119,10 +128,48 @@ def meta_none(self) -> Optional[TypesBase.meta]:
         Returns default for meta data if it is None.
 
         Returns:
-            Optional[np.ndarray]: Default for meta data.
+            Optional[TypesBase.meta]: Default for meta data.
         """
         return None
 
+    def meta_auto(self, idx: int) -> TypesBase.meta:
```

Comment:
let's move this to `get_meta` as we want them to available for all dataloaders by default. 
That means we should change all daughter class `get_meta` in order to append the dataset-specific meta-data to these default ones.

Reply:
Done